src/common/SeasonalNotification/SeasonalNotification.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2017-2023 Smart code 203358507
+
+const React = require('react');
+const styles = require('./styles');
+const { Button, ModalDialog } = require('../../common');
+const useBinaryState = require('stremio/common/useBinaryState');
+const useFetchNotificationData = require('./useFetchNotificationData');
+
+const SeasonalNotification = () => {
```

Comment:
Should be name `SeasonalModal` or better `EventModal`

Reply:
will change the naming